### DISCUSSION TITLE
Reason for not receiving email

### DISCUSSION THREAD

**Discussion by AABehzad** [PHPMailer/PHPMailer#2350] [General]
Hello, which one is as follows, but whatever I do, I do not receive the email, maybe there is a problem with my server ?? But the PHPmailer function returns the TRUE value 
`					
					$phpmailer = new PHPMailer();
					$phpmailer->isSMTP();
					$phpmailer->Host = 'smtp.mailtrap.io';
					$phpmailer->SMTPAuth = true;
					$phpmailer->Port = 2525;
					$phpmailer->Username = Secret;
					$phpmailer->Password = Secret;
					$phpmailer->setFrom($email,$name);
					$phpmailer->addAddress("ajdar9667@gmail.com");
					$phpmailer->CharSet= "UTF-8";
					$phpmailer->isHTML(true);                                 
					$phpmailer->Subject = $subject ;
					$phpmailer->Body   = $text ;
					$t = $phpmailer->send();`
@bd808 @freephile @mscdex @eexit

**Comment by aleksandrmelnikov** [comment]
Have you tried setting this parameter and looking at the output, after your test?
`$​phpmailer->​SMTPDebug​ = ​SMTP​::​DEBUG_SERVER​;`

**Reply by AABehzad** [reply]
I put this code you said but it did not work !!

**Reply by AABehzad** [reply]
I use the mailtrap service and when I send the message, the message is confirmed by the mailtrap server, and I can even see it in my account on the mailtrap site, but it is not received in my Gmail account. 
@aleksandrmelnikov

**Comment by Synchro** [comment]
You have no error checking at all, so you won't know when things are not working.

Enabling debug output won't change how anything works, but it will show you what is happening in detail.

If your from address is a gmail address and you're sending it through mailtrap, gmail will reject it because it's forgery, and is not permitted by Google's SPF. Mailtrap should be spotting this and rejecting your messages.

**Reply by AABehzad** [reply]
Do you mean that I do not use the mailtrap service?  @Synchro

**Reply by Synchro** [reply]
Not if you want to send from a gmail address.
